src/MoBi.Presentation/Presenter/EditContainerPresenter.cs

```diff
@@ -29,7 +32,7 @@ public interface IEditContainerPresenter : ICanEditPropertiesPresenter, IPresent
       void UpdateParentPath();
       string ContainerModeDisplayFor(ContainerMode mode);
       IReadOnlyList<ContainerMode> AllContainerModes { get; }
-      void SetContainerMode(ContainerMode newContainerMode);
+      bool SetContainerMode(ContainerMode newContainerMode);
```

Comment:
could you please add a comment explaining what the boolean return value means? (not obvious from the function name)